config/remote/remote.go

```diff
@@ -212,23 +233,32 @@ func findMatchingRecord(ss []ParamValue, ver string, rollout int) (match *ParamV
 	}
 	// as a last step, check if app's rollout group matches feature's rollout value
 	// (do not try to use other match with lesser weight)
-	if match != nil && match.Rollout > rollout {
-		match = nil
+	if match != nil {
+		if match.Rollout > c.rolloutGroup {
+			c.analytics.EmitPartialRolloutEvent(ClientCli, featureName, match.Rollout, partialRolloutPerformedFailure)
+			match = nil
+		} else {
+			c.analytics.EmitPartialRolloutEvent(ClientCli, featureName, match.Rollout, partialRolloutPerformedSuccess)
+		}
+	} else {
+		//when there's no match (eg., due to a version value mismatch) emit the partial rollout event failure
+		//TODO: change the structure of this event to hold error anyway?
```

Comment:
Leftover or something for other PR?

Reply:
Note for this branch, where the match is nil we must emit _some_ value of feature rollout group (now made it zero, arbitrarily)  - this is why I added this `TODO` here. So we're either fine about it, or an future update might be needed in here.